arangopipe/tests/AQL/query_1.json

```diff
@@ -0,0 +1 @@
+[{"value":"FOR d IN datasets\n  FILTER d.name == 'wine dataset'\n  REMOVE d IN datasets","parameter":{},"name":"query_1"},{"value":"FOR f in featuresets\n    FILTER f.name == 'wine_no_transformations'\n        REMOVE f IN featuresets","parameter":{},"name":"query_2"},{"value":"INSERT {\"name\" : \"wine dataset\",\n                   \"description\": \"Wine quality ratings\",\"source\": \"UCI ML Repository\" } INTO datasets","parameter":{},"name":"query_3"},{"value":"INSERT {\"fixed acidity\": \"float64\",\n               \"volatile acidity\": \"float64\",\n               \"citric acid\": \"float64\",\n               \"residual sugar\": \"float64\",\n               \"chlorides\": \"float64\",\n               \"free sulfur dioxide\": \"float64\",\n               \"total sulfur dioxide\": \"float64\",\n               \"density\": \"float64\",\n               \"pH\": \"float64\",\n               \"sulphates\": \"float64\",\n               \"alcohol\": \"float64\",\n               \"quality\": \"int64\",\n               \"name\": \"wine_no_transformations\"} INTO featuresets","parameter":{},"name":"query_4"},{"value":"FOR ds IN datasets\n    FILTER ds.name == 'wine dataset'\n        FOR fs in featuresets\n            FILTER fs.name == 'wine_no_transformations'\n            INSERT {\"_from\": fs._id, \"_to\": ds._id} INTO featureset_dataset","parameter":{},"name":"query_5"}]
```

Comment:
Does it make sense to have a markdown file describing these queries? Or am I missing this file (as you have the graphics...)

Reply:
Yes, good idea.